Hannah Rosenbaum Final Project Part 1

hrosenbaum opened the pull request.

RosemaryPang committed:
change the date

hrosenbaum committed:
FinalPart1

hrosenbaum committed:
FinalPart1

SidharthSaluja reviewed (commented):
1) Can you remove _freeze/posts/HW3_Solutions_Pang/ (from freeze folder)
2) Can you remove HW3_solutions_Pang.qmd in posts folder. 
3) Can you dont have space between file names  so rename the file -(FinalProjectPart1_{Your_name}). Be sure to enter your name.
4) Can you render FinalProjectPart1_{Your_name} to generate freeze files

hrosenbaum committed:
Removed freeze/posts/HW3_Solutions_Pang
Removed HW3_Solutions_Pang
Added name to Final Project Part 1 qmd
Rerendered final project part 1

hrosenbaum committed:
Rerendered FinalProjectPart1_HannahRosenbaum

SidharthSaluja commented:
Umm. I still get merge conflict. I slacked you the issue - Why you forked Rosemary repo instead of 603. I am closing this for now
